packages/FileFormatter/Formatter/JsonFileFormatter.php

```diff
@@ -17,7 +17,7 @@
 final class JsonFileFormatter implements FileFormatterInterface
 {
     public function __construct(
-        private PrinterInterface $jsonPrinter
+        private PrinterInterface $printer
```

Comment:
This is similar false postive case as the previous one. It must be kept too

Reply:
implemented https://github.com/rectorphp/rector-src/pull/168/commits/f0ea6d31c404e0793529ba0749d89a4b4e945f6e